test/integration/stack_test.go

```diff
@@ -0,0 +1,55 @@
+//go:build integration
+// +build integration
+
+package integration
+
+import (
+	"testing"
+)
+
+func TestStack(t *testing.T) {
+	// Setup
+	client := getClient(t)
+	var err error
+
+	_, err = client.GetStackById(-999)
+	if err == nil {
+		t.Fatal("Expected GetStackById to return an error but got none!")
+		return
+	}
+
+	individualResult, err := client.GetStackByName("THIS_STACK_SHOULD_NOT_EXIST_EVER")
+	if err == nil {
+		t.Fatal("Expected GetStacksByName to return an error but did not get an err!")
```

Comment:
The name of the method is `GetStackByName` (singular) now.

```suggestion
		t.Fatal("Expected GetStackByName to return an error but did not get an err!")
```